### ISSUE TITLE
Where can I find data_extraction.py file?

### ISSUE THREAD

**Issue by sejal-varu** [rajaharsha/AmazonProductRecommendation-CF-ALS-Spark#1]

**Comment by rajaharshachinta** [comment]
I don't think we require this file. Where did you find data_extraction.py?

**Comment by sejal-varu** [comment]
Seems you performed some ETL on original data, as I wanted to run same for other categories too. Hence, need for data-extraction.py

**Comment by rajaharshachinta** [comment]
Yeah, I used python to prepare data in the required input format. 

But, I do not have that file now. I will check and upload in-case I find it.
